### ISSUE TITLE
Archive this repo?

### ISSUE THREAD

**Issue by traversaro** [robotology/gym-ignition-models#36]
Given that we already archived https://github.com/robotology/gym-ignition, without any hurry I think it would be great if we could archive also this repo, so we have a repo less to worry about.

fyi @diegoferigo @CarlottaSartore @GiulioRomualdi @flferretti @xela-95 @Giulero

**Comment by diegoferigo** [comment]
Definitely, this repo can be archived now. We should double check if there are repos still using it and migrate to alternatives.

**Comment by diegoferigo** [comment]
@traversaro archiving the repo now.
